src/main/java/com/bmc/truesight/saas/meter/client/model/Event.java

```diff
@@ -34,20 +42,42 @@ public Instant timestamp() {
     public abstract Optional<String> at();
```

Comment:
Can we javadoc the fields directly, and change the method signature for at() & ad() (eg instead of ad(), it should be appData())